examples/manipulator/children.py

```diff
@@ -1,20 +1,19 @@
-import math
-import os
-
+from kivy.app import App
 from kivy.lang import Builder
-from kivy.properties import ListProperty, ObjectProperty
+from kivy.properties import ListProperty
+from kivy.properties import ObjectProperty
 from kivy.uix.boxlayout import BoxLayout
-from kivy.uix.slider import Slider
-
-import kivy3
-from kivy.app import App
-from kivy.uix.floatlayout import FloatLayout
-from kivy.clock import Clock
-from kivy3 import Mesh, Material
-from kivy3 import Scene, Renderer, PerspectiveCamera
+from kivy3 import Material
+from kivy3 import Mesh
+from kivy3 import PerspectiveCamera
+from kivy3 import Renderer
+from kivy3 import Scene
 from kivy3.extras.geometries import BoxGeometry
+import math
+import os
+
+"""Demonstrate relative transform of children of a Mesh object."""
```

Comment:
I think module level docstring are usually on the top of the file (before the imports).
https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring

Reply:
merged docsstring with example app docstring